packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -233,80 +233,87 @@ const makeRenderer = (
 	const host = `${requestProtocol}://${domain}`;
 	const userAgent = headers['user-agent'];
 	const userAgentDevice = headers['x-ua-device'] || ''; // set by fastly
+	const memberId = parseMemberCookie(state).id;
 
 	// create the store with populated `config`
-	const initialState = {
-		config: {
-			apiUrl: API_ROUTE_PATH,
-			baseUrl: host,
-			enableServiceWorker,
-			requestLanguage,
-			supportedLangs,
-			initialNow: new Date().getTime(),
-			variants: getVariants(state),
-			entryPath: url.pathname, // the path that the user entered the app on
-			media: getMedia(userAgent, userAgentDevice),
-		},
-	};
+	const initializeStore = () =>
+		request.server['mwp-app-route-plugin'].getFlags(memberId).then(flags => {
```

Comment:
what if getFlags fails?

Reply:
good call - will add a `catch`

Reply:
I've moved the error handling into the `getFlags` function itself - it is now guaranteed to return an object (even if the object is empty because the flags couldn't be read for some reason)